app/controllers/support_interface/zendesk_controller.rb

```diff
@@ -36,6 +39,19 @@ def destroy
       end
     end
 
+    def export
+      @zendesk_delete_requests = ZendeskDeleteRequest.all
+      respond_to do |format|
+        format.csv do
+          response.headers["Content-Type"] = "text/csv"
+          response.headers[
+            "Content-Disposition"
+          ] = "attachment; filename=#{Time.zone.now.iso8601}_zendesk_delete_tickets.csv"
+          render :export
+        end
+      end
```

Comment:
```suggestion
      filename = "#{Date.today.iso8601}_deleted_zendesk_tickets.csv"
      respond_to do |format|
        format.csv do
          response.headers["Content-Type"] = "text/csv"
          response.headers[
            "Content-Disposition"
          ] = "attachment; filename=#{filename}"
          render :export
        end
      end
```

Reply:
@tvararu as per your suggested, this will only return the date. Did we want a timestamp or just the date?